contrib/openbsm/bin/bsmconv/linau_field.c

```diff
@@ -0,0 +1,196 @@
+#include <ctype.h>
+#include <nv.h>
+#include <string.h>
+#include <stdlib.h>
+
+#include "linau_field.h"
+#include "linau_impl.h"
+#include "pjdlog.h"
+
+static size_t find_string_value_end(const char *buf, size_t start,
+    char stringtype);
+
+
+static size_t
+find_string_value_end(const char *buf, size_t start, char stringtype)
```

Comment:
I'm not saying you shouldn't use find_string_value_end. Just use strstr internally in this function.